component_tabs.yaml

```diff
@@ -21,6 +21,8 @@ patterns/grid:
   accessibility: True
 patterns/heading-icon:
   accessibility: True
+patterns/hero:
```

Comment:
We should not mix up these tabs with the new implementation (I think). Unless we change it to be more flexible.

These tabs were built to support 3 items:
- implementation (always present by default)
- design guidelines
- accessibility

If we use tabs for patterns we probably need a completely different set of tabs: one tab per layout/variant.
And within that tab, a second level of segmented control switches.

The current example makes it quite confusing, because it mixes us the old "Implementation" tab with new layout based tab, making it unclear what we want to achieve.